include/cute/atom/copy_traits_xe.hpp

```diff
@@ -536,7 +629,25 @@ struct Copy_Traits_<XE_2D_U8x16x32_LD_N, args_t...>
 
   template <class... ArgT>
   Copy_Traits_(ArgT... args)
-      : XE_2D_LD_Unpack<XE_2D_U8x16x32_LD_N, args_t...>(args...) {}
+      : XE_2D_LD_Unpack<XE_2D_Packed_U8x16x32_LD_N, args_t...>(args...) {}
+};
+
+template <class... args_t>
+struct Copy_Traits_<XE_2D_Packed_U8x32x32_LD_N, args_t...>
+    : XE_2D_LD_Unpack<XE_2D_Packed_U8x32x32_LD_N, args_t...> {
+  using ThrID = Layout<_16>;
+  // Map from (src-thr,src-val) to bit
+  using SrcLayout = Layout<Shape <_16,Shape <_8,  _2, _32>>,
+                           Stride< _0,Stride< _1,_128,_256>>>;
+  // Map from (dst-thr,dst-val) to bit
+  using DstLayout = Layout<Shape <_16,Shape <_8,  _2, _32>>,
+                           Stride<_16,Stride< _1,_128,_256>>>;
```

Comment:
```suggestion
  using SrcLayout = Layout<Shape <_16,Shape <_8,  _2, _32>>,
                           Stride< _0,Stride< _1,_2,_256>>>;
  // Map from (dst-thr,dst-val) to bit
  using DstLayout = Layout<Shape <_16,Shape <_8,  _2, _32>>,
                           Stride<_16,Stride< _1,_2,_256>>>;
```

Reply:
Sorry, I had assumed this one meant that a work-item will copy 2 columns, both separated by 16 columns (so a workitem's two elements corresponding to the same row of the 2D block would be 128 bits apart, and a work-item's next element in the next column 2D block will be 256 bits apart).

Can you please explain how these layouts are interpreted? Thanks

Reply:
```suggestion
  using SrcLayout = Layout<Shape <_16,Shape <_8,  _2, _32>>,
                           Stride< _0,Stride< _1,_128,_256>>>;
  // Map from (dst-thr,dst-val) to bit
  using DstLayout = Layout<Shape <_16,Shape <_8,  _2, _32>>,
                           Stride<_8,Stride< _1,_128,_256>>>;
```
In that case I believe just the stride between workitems needs to be updated.

Reply:
Thanks for the info, @t4c1!

Please help resolve another confusion - since this layout is for a packed U8 copy atom, does a stride of `8` mean that the elements read by work-item 0 & work-item 1 are 8 bits apart (so, adjacent columns)? But then in packed copy, work-items are supposed to copy 2 consecutive columns?

Reply:
That is exactly right.

Reply:
Thanks again, @t4c1!

Sorry for the confusion! My question was a bit ambiguous - the copy documentation PR mentions that each work-item in packed copy atoms copies two consecutive columns, so `work-item 0` would copy `columns 0 & 1`.

However, it seems that the `DstLayout` here suggests that `work-item 0` would copy `columns 0 & 16`.

If that's correct, the description of the copy documentation PR seems contradictory.

Can you please clarify? Thanks!

Reply:
I am not sure what does this copy do - you can check using copy_debug utility I mentioned earlier. Previously I assumed you knew, but I am not so sure now.

Reply:
> Previously I assumed you knew

I did not. Will check out the utility & report back. Thanks!